src/openshift/openshiftItem.ts

```diff
@@ -55,6 +55,18 @@ export abstract class OpenShiftItem {
         return (validator.matches(value, '^[a-z]([-a-z0-9]*[a-z0-9])*$')) ? null : message;
     }
 
+    static hideTokenpassword(value: string) {
+        const regex = /--token\s*=\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
+        const tokenRegex = value.match(regex);
+        return (tokenRegex) ?  value.replace(tokenRegex[1], '**********') : value;
+    }
+
+    static hideCredentialPassword(value: string) {
+        const regex = /-p\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
+        const tokenRegex = value.match(regex);
+        return (tokenRegex) ? value.replace(tokenRegex[0], '-p **********'): value;
```

Comment:
You could replace just the first group and get rid of the '-p' like in the token masking method.

Reply:
@jrichter1  if I use the tokenRegex[1], remove the -p from my regex for replacing and if there are two same string  for example 
```
username: abc
password: abc
then the output would be 
`login https://192.168.64.74:8443 -u ***(which is abc) -p abc --insecure-skip-tls-verify`
it will replace the first string match.
```
that is why I am using the `-p and password ` so that it will update the only password.

Reply:
I meant keep the -p part of the regex, use tokenRegex[1] and replace the match with only asterisks. That shouldn't cause this sort of problem.

Reply:
If I do `value.replace(tokenRegex[1], '-p **********')`
```
username: abc
password: abc
then the output would be 
login https://192.168.64.74:8443 -u -p ***(which is abc) -p abc --insecure-skip-tls-verify
```

Reply:
nevermind, I kept thinking about the regex I totally forgot replace takes the first occurence of a string

Reply:
Lets keep it simple:

``` JavaScript
export class Filters {
    static readonly tokenRegex = /--token\s*=\s*([^\s]*)/;

    static filterToken(value: string) {
        return value.replace(Filters.tokenRegex, '--token **********');
    }

    static readonly passwordRegex = /-p\s*(.*)\s/;

    static filterPassword(value: string) {
        return value.replace(Filters.passwordRegex, '-p **********');
    }
}
```